tests/_utils-tests/editor.js

```diff
@@ -0,0 +1,56 @@
+/**
```

Comment:
Should we create tests for mocks?

Reply:
Why not? This test ensures API of the Editor mock which is used in other tests.